### PR TITLE
[release-1.23] Update OCP max version to 4.10

### PR DESCRIPTION
**Pull request by dsimansk**
Per title. Keeping the metadata updated. :)

I'll update `main` shortly.

/cc @mgencur @matzew @skonto

### CODE REVIEW

**Comment by skonto** [comment]
Upgrade tests are failing a lot lately.

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ spec:
 
     The components provided with the OpenShift Serverless operator require minimum cluster sizes on
     OpenShift Container Platform. For more information, see the documentation on [Getting started
-    with OpenShift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/install#install-serverless-operator).
+    with OpenShift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/serverless/install#install-serverless-operator).
```
[the remark this answers is not in the document]
Comment (dsimansk) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:87]: Fixed in https://github.com/openshift-knative/serverless-operator/pull/1595/commits/a5f83468e81e604516f3dda9b249462949ef6702.

**Comment by dsimansk** [comment]
Why not to retest again, Mr. Prow?! :)

/shrug

**Pull request merged**